src/assets/TokenRatesController.test.ts

```diff
@@ -57,12 +65,15 @@ describe('TokenRatesController', () => {
       .reply(200, {})
       .get(`${COINGECKO_BSC_PATH}?contract_addresses=0xfoO&vs_currencies=gno`)
       .reply(200, {})
-
+      .get(COINGECKO_SUPPORTED_CURRENCIES)
+      .reply(200, ['eth', 'usd'])
       .persist();
 
     nock('https://min-api.cryptocompare.com')
       .get('/data/price?fsym=ETH&tsyms=USD')
       .reply(200, { USD: 179.63 })
+      .get('/data/price?fsym=ETH&tsyms=ETH')
```

Comment:
Interesting :thinking: Are we asking CryptoCompare to convert ETH to ETH?

Reply:
Can't remember why I added this... Doesn't seem to be necessary! Will remove!